docs/src/operations/index.rst

```diff
@@ -3,18 +3,37 @@
 Operations
 ==========
 
+.. py:currentmodule:: metatensor
+
 The Python API for metatensor also provides functions which operate on
-:py:class:`metatensor.TensorMap`, :py:class:`metatensor.TensorBlock`, and
-:py:class:`metatensor.Labels` and can be used to build your own Machine Learning
-models. These operations are provided in the ``metatensor-operations`` python
-package, which is installed by default when doing ``pip install metatensor``.
+:py:class:`TensorMap`, :py:class:`TensorBlock`, and :py:class:`Labels` and can
+be used to build your own Machine Learning models. These operations are provided
```

Comment:
is there a reason to capitalise Machine Learning like this?

Reply:
not really, I'll change it!